test/exports.spec.js

```diff
@@ -1,15 +1,14 @@
-import * as chai from 'chai'
-import * as path from 'path';
-import * as glob from 'glob';
-import toCamelCase from 'camelcase';
-import interopRequire from 'interop-require';
-import * as decorators from '../';
+const chai = require('chai');
+const path = require('path');
+const glob = require('glob');
+const camelCase = require('camelCase');
+const interopRequire = require('interop-require');
+const decorators = require('core-decorators');
```

Comment:
@BurtHarris I wasn't going to burden you _right now_ with it, but I was hoping to also switch the tests over to TypeScript so that we're actually also testing the type definitions at the same time. This can come later (if ever). For now your plan is OK with me.